core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -547,19 +547,19 @@ private StartupConsistencyChecker(Long fileId) {
   /**
    * Class to represent the status and result of the startup consistency check.
    */
-  public static final class StartupConsistencyCheckResult {
+  public static final class StartupConsistencyCheck {
     /** Result for a disabled check. */
-    private static final StartupConsistencyCheckResult DISABLED =
-        new StartupConsistencyCheckResult(Status.DISABLED, null);
+    private static final StartupConsistencyCheck DISABLED =
```

Comment:
optional: I would inline these static fields as long as they are only used in one place